### DISCUSSION TITLE
what happen if i emplace same component multi times in same entity？

### DISCUSSION THREAD

**Discussion by armorhu** [skypjack/entt#812] [General]
eg. 
reg->emplace<CompA>(entity);
reg->emplace<CompA>(entity);
reg->emplace<CompA>(entity);
reg->emplace<CompA>(entity);

reg->view<CompA>([](CompA& value)
{
   // how many times call?
  // call once. value = first comp A？
  // call once ，value = last comp a？
  // call forth
}
)

**Comment by skypjack** [comment]
It triggers an assert the second time you try to emplace the same component. That's it.

**Reply by armorhu** [reply]
![image](https://user-images.githubusercontent.com/4246258/147028177-3ba07e89-bd87-49a1-90d4-a1dd578fed1d.png)

I actually do this success without any assert.

**Reply by skypjack** [reply]
Tests explicitly verify this and it's pretty easy to reproduce actually. See [here](https://godbolt.org/z/8cq689Ys1) for an example.
Remember that `ENTT_ASSERT` forwards the request to `assert` by default, therefore it doesn't work in release mode. Is this the case?
